protobuf/mayastor.proto

```diff
@@ -500,6 +501,26 @@ message RebuildStatsReply {
   uint64 tasks_active = 7; // number of current active tasks
 }
 
+message RebuildHistoryRequest {
+  string uuid = 1;  // uuid of the nexus
+}
+
+message RebuildHistoryReply {
+  string nexus = 1;                          // Name of the nexus
+  repeated RebuildHistoryRecord records = 2; // List of all rebuild records
+                                             // across all the replicas
+}
+
+message RebuildHistoryRecord {
+  string dst_uri = 1;            // URI of rebuilt replica (destination)
+  string src_uri = 2;            // URI of source replica
+  string state = 3;              // Final state of rebuild job
```

Comment:
Can this be an enum?

Reply:
Followed from RebuildStateReply. I don't see an apparent benefit to make it enum as this is going to be the eventual state of RebuildJob.

Reply:
If we want to use it an enum allows us to check for specific states. Otherwise just by reading this api I cannot tell which state exist? 
Having the enum will also mean the autogenerated code will have the type nicely defined and easy to use.